tls/s2n_signature_algorithms.c

```diff
@@ -115,6 +115,20 @@ int s2n_get_and_validate_negotiated_signature_scheme(struct s2n_connection *conn
         }
     }
 
+    /* We require an exact match in TLS 1.3, but all previous versions can fall back to the default SignatureScheme.
+     * This means that an s2n client will accept the default SignatureScheme from a TLS server, even if the client did
+     * not send it in it's ClientHello. This pre-TLS1.3 behavior is an intentional choice to maximize support. */
```

Comment:
Does this comment need to be updated since the behavior of this function is changing? https://github.com/awslabs/s2n/blob/8123d4788ad5c468af61725e34a235ab03c2bab8/tls/s2n_server_key_exchange.c#L57

Reply:
Also as a note, I think this comment might be wrong (saying the Server picked instead of the client): https://github.com/awslabs/s2n/blob/f2e3cbe9841d316ca2f8fd9abc3bfc1987157246/tls/s2n_client_cert_verify.c#L36

Reply:
Fixed